ngs/dogs-and-cats/README.md

```diff
@@ -0,0 +1,17 @@
+# DogsAndCats Actor
+
+This actor implements a simple piece of functionality: source and display a random image of either a cat or a dog.
+
+This actor uses the following capabilities:
+1. `wasmcloud:httpserver` to receive HTTP requests
+1. `wasmcloud:httpclient` to make HTTP requests to public APIs for animal image URLs, and then query that URL for the picture
+1. `wasmcloud:builtin:numbergen` to generate a random number to decide if a cat or dog shoud be fetched
+
+The following commands can be run to start this actor, two capability provider implementations that satisfy the contracts, and link them together. Note that the actor public key here (starting with `MCUCZ`) assumes you're using the wasmCloud published version of this actor.
+```bash
```

Comment:
Do we want to add a warning here that if people are starting these actors from local builds their public keys will be different?

Reply:
Yeah that's what I was going for with the last sentence there, but I will clarify that users will have to change it

Reply:
Huh I didn't even notice that there.